substrate/frame/support/procedural/src/lib.rs

```diff
@@ -1371,6 +1371,25 @@ pub fn whitelist_storage(_: TokenStream, _: TokenStream) -> TokenStream {
 	pallet_macro_stub()
 }
 
+/// The optional attribute `#[pallet::disable_try_decode_storage]` will declare the
+/// storage as whitelisted from decoding during try-runtime checks. This should only be
+/// attached to transient storage which cannot be migrated during runtime upgrades.
+///
+/// ### Example
+/// ```ignore
+/// #[pallet::storage]
+/// #[pallet::disable_try_decode_storage]
+/// pub(super) type Events<T: Config> = StorageValue<_, Vec<Box<EventRecord<T::RuntimeEvent, T::Hash>>>, ValueQuery>;
+/// ```
```

Comment:
No more `ignore` docs please 🙈 a real pallet requires just a few extra lines of boilerplate.

Reply:
This seems to be impossible in the `frame-support-procedural` crate, because depending on anything `frame` related results in cyclic deps.